LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,16 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail MIQ Logs:
+function tailmiq()
+{
+  tail -f /var/www/miq/vmdb/log/$1.log
```

Comment:
Originally I submitted this with each individual log file split out and the feedback was to make it generic.  e.g. 

alias auto='tail -f /var/www/miq/vmdb/log/automation.log'
alias evm='tail -f /var/www/miq/vmdb/log/evm.log'
alias aws='tail -f /var/www/miq/vmdb/log/aws.log'
...

Regardless I think we need more than just evm.log